modules/serverless-cluster-sizing-req.adoc

```diff
@@ -1,7 +1,3 @@
-// Module included in the following assemblies:
```

Comment:
This was more recently discussed (last week I think?) in a core OCP team meeting and consensus was to keep it because most people found it helpful (and it's still in the checklist).